FrostySdk/Managers/Infos/FileInfos/NonCasFileInfo.cs

```diff
@@ -22,13 +22,18 @@ public class NonCasFileInfo : IFileInfo
     private readonly string m_fullPath;
     private readonly string? m_fullBasePath;
 
+    private readonly bool m_fullPathExists;
+    private readonly bool m_fullBasePathExists;
```

Comment:
I think its better to make this nullable
```suggestion
    private readonly bool? m_fullBasePathExists;
```